angular/e2e/feedback/feedback.e2e-spec.ts

```diff
@@ -9,18 +17,25 @@ describe('Feedback', () => {
     page = new FeedbackPage();
   });
 
+  for(let i=0; i<csvValues.data.length-1; i++) {
+    let j = 0;
   // Check if Feedback-Page is loaded
   it('should load Home-Page ', () => {
     page.navigateToFeedback();
     expect(page.getParagraphText()).toEqual('Feedback');
   });
 
   // Check if textfield appears after star-rating, submit and check if it is approved
-  it('feedback page opens and elements are dynamically', () => {
-    page.navigateToFeedback();
-    page.setStarRating();
-    expect(element(page.getTextField).isDisplayed()).toBe(true);
-    page.submitFeedback();
-    expect(element(page.getThanks).isDisplayed()).toBe(true);
-  });
+    it('feedback page opens and elements are dynamically', () => {
```

Comment:
Die Testdatenbefüllung sollte in einen eigenen Test ausgelagert werden. Der vorherige Test macht weiterhin Sinn und sollte daher glaube ich bestehen bleiben

Reply:
Autsch, ich hatte es aus Versehen beseitigt, denn dieser Abschnitt war als Kommentar eingetragen, als ich Testvorgang beschleunigen wollte.

Reply:
besteht weiterhin